src/components/Coaching/LoadCoachingList.test.tsx

```diff
@@ -0,0 +1,52 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { GqlMockedProvider } from '../../../__tests__/util/graphqlMocking';
+import { useLoadCoachingListQuery } from './LoadCoachingList.generated';
+
+describe('LoadCoaching', () => {
+  it('query correct', async () => {
+    const { result, waitForNextUpdate } = renderHook(
+      () => useLoadCoachingListQuery(),
+      {
+        wrapper: GqlMockedProvider,
+      },
+    );
```

Comment:
I think that the test should be testing rendering the component and not testing the query alone. If the component renders, and we can query the DOM for the data we expect then that determines that the query ran successfully.